### PR TITLE
[Hotfix 4.0.1] Release notes

### PR DESCRIPTION
**Pull request by JRahnama**

### CODE REVIEW

**Comment by johnnypham** [comment]
The changes aren't up-to-date with #1474. Also, some of my suggestions weren't reviewed in that PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## Microsoft.Data.SqlClient 4.0.1 released 14 January 2022
```
Comment (DavoudEshtehari) [inline CHANGELOG.md:7]:
```suggestion
## Microsoft.Data.SqlClient 4.0.1 released 17 January 2022
```

File: release-notes/4.0/4.0.1.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 4.0.1 released 14 January 2022
```
Comment (DavoudEshtehari) [inline release-notes/4.0/4.0.1.md:3]:
```suggestion
## Microsoft.Data.SqlClient 4.0.1 released 17 January 2022
```

File: release-notes/4.0/4.0.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ The following Microsoft.Data.SqlClient 4.0 stable releases have been shipped:
 
 | Release Date | Version | Notes |
 | :-- | :-- | :--: |
+| 2022/01/22 | 4.0.1 | [release notes](4.0.1.md) |
```
Comment (DavoudEshtehari) [inline release-notes/4.0/4.0.md:7]:
```suggestion
| 2022/01/17 | 4.0.1 | [release notes](4.0.1.md) |
```

File: release-notes/4.0/README.md [hunk 4]
```diff
@@ -4,6 +4,7 @@ The following Microsoft.Data.SqlClient 4.0 stable releases have been shipped:
 
 | Release Date | Version | Notes |
 | :-- | :-- | :--: |
+| 2022/01/22 | 4.0.1 | [release notes](4.0.1.md) |
```
Comment (DavoudEshtehari) [inline release-notes/4.0/README.md:7]:
```suggestion
| 2022/01/17 | 4.0.1 | [release notes](4.0.1.md) |
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
...
+
+This update brings the below changes over the previous preview release:
+
+### Added
+
+- AppContext switch `SuppressInsecureTLSWarning` is added to make users to be able to supress TLS security warning while `Encrypt=false` on connection string. [#1467](https://github.com/dotnet/SqlClient/pull/1467) 
+
+### Bug Fixes
+
+- Fixed the issue with Kerberos authentication using .NET 6. [#1468](https://github.com/dotnet/SqlClient/pull/1468)
```
Comment (johnnypham) [inline CHANGELOG.md:17]:
```suggestion
- Fixed Kerberos authentication failure when using .NET 6. [#1468](https://github.com/dotnet/SqlClient/pull/1468)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## Microsoft.Data.SqlClient 4.0.1 released 17 January 2022
+
+This update brings the below changes over the previous preview release:
+
+### Added
+
+- AppContext switch `SuppressInsecureTLSWarning` is added to make users to be able to supress TLS security warning while `Encrypt=false` on connection string. [#1467](https://github.com/dotnet/SqlClient/pull/1467)
```
Comment (johnnypham) [inline CHANGELOG.md:13]:
```suggestion
- Added AppContext switch `SuppressInsecureTLSWarning` to allow suppression of TLS security warning when using `Encrypt=false` in the connection string.[#1467](https://github.com/dotnet/SqlClient/pull/1467) 
```

File: release-notes/4.0/4.0.1.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+This update brings the below changes over the previous preview release:
+
+### Added
+
+- AppContext switch `SuppressInsecureTLSWarning` is added to make users to be able to supress TLS security warning while `Encrypt=false` on connection string. [#1467](https://github.com/dotnet/SqlClient/pull/1467) [Read more](#suppress-tls-security-warnings)
+
+### Bug Fixes
+
+- Fixed the issue with Kerberos authentication using .NET 6. [#1468](https://github.com/dotnet/SqlClient/pull/1468)
```
Comment (DavoudEshtehari) [inline release-notes/4.0/4.0.1.md:13]:
```suggestion
- Fixed Kerberos authentication failure when using .NET 6. [#1468](https://github.com/dotnet/SqlClient/pull/1468)
```

File: release-notes/4.0/4.0.1.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 4.0.1 released 17 January 2022
+
+This update brings the below changes over the previous preview release:
+
+### Added
+
+- AppContext switch `SuppressInsecureTLSWarning` is added to make users to be able to supress TLS security warning while `Encrypt=false` on connection string. [#1467](https://github.com/dotnet/SqlClient/pull/1467) [Read more](#suppress-tls-security-warnings)
```
Comment (DavoudEshtehari) [inline release-notes/4.0/4.0.1.md:9]:
```suggestion
- Added AppContext switch `SuppressInsecureTLSWarning` to allow suppression of TLS security warning when using `Encrypt=false` in the connection string. [#1467](https://github.com/dotnet/SqlClient/pull/1467) [Read more](#suppress-tls-security-warnings)
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## Microsoft.Data.SqlClient 4.0.1 released 17 January 2022
```
Comment (Kaur-Parminder) [inline CHANGELOG.md:7]: ## Microsoft.Data.SqlClient 4.0.1 released 2022-01-17

File: CHANGELOG.md [hunk 10]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
...
+### Added
+
+- Added AppContext switch `SuppressInsecureTLSWarning` to allow suppression of TLS security warning when using `Encrypt=false` in the connection string.[#1467](https://github.com/dotnet/SqlClient/pull/1467) 
+
+### Bug Fixes
+
+- Fixed Kerberos authentication failure when using .NET 6. [#1468](https://github.com/dotnet/SqlClient/pull/1468)
+- Fixed the issue with `SqlLocalDB` instance pipe name. [#1475](https://github.com/dotnet/SqlClient/pull/1475)
+- Fixed the issue with enclave providers not being thread safe. [#1466](https://github.com/dotnet/SqlClient/pull/1466)
+- Updated obsolete API calls in NET6. [#1471](https://github.com/dotnet/SqlClient/pull/1471)
```
Comment (DavoudEshtehari) [inline CHANGELOG.md:20]:
```suggestion
- Fixed connection failure when using `SqlLocalDB` instance pipe name. [#1475](https://github.com/dotnet/SqlClient/pull/1475)
- Fixed a failure when executing concurrent queries requiring enclaves. [#1466](https://github.com/dotnet/SqlClient/pull/1466)
- Updated obsolete API calls targeting .NET 6. [#1471](https://github.com/dotnet/SqlClient/pull/1471)
```

File: release-notes/4.0/4.0.1.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+### Added
+
+- Added AppContext switch `SuppressInsecureTLSWarning` to allow suppression of TLS security warning when using `Encrypt=false` in the connection string. [#1467](https://github.com/dotnet/SqlClient/pull/1467) [Read more](#suppress-tls-security-warnings)
+
+### Bug Fixes
+
+- Fixed Kerberos authentication failure when using .NET 6. [#1468](https://github.com/dotnet/SqlClient/pull/1468)
+- Fixed the issue with `SqlLocalDB` instance pipe name. [#1475](https://github.com/dotnet/SqlClient/pull/1475)
+- Fixed the issue with enclave providers not being thread safe. [#1466](https://github.com/dotnet/SqlClient/pull/1466)
+- Updated obsolete API calls in NET6. [#1471](https://github.com/dotnet/SqlClient/pull/1471)
```
Comment (DavoudEshtehari) [inline release-notes/4.0/4.0.1.md:16]:
```suggestion
- Fixed connection failure when using `SqlLocalDB` instance pipe name. [#1475](https://github.com/dotnet/SqlClient/pull/1475)
- Fixed a failure when executing concurrent queries requiring enclaves. [#1466](https://github.com/dotnet/SqlClient/pull/1466)
- Updated obsolete API calls targeting .NET 6. [#1471](https://github.com/dotnet/SqlClient/pull/1471)
```

**Review by Kaur-Parminder — APPROVED** [review]

**Pull request merged**